src/views/InteractionDesigner.vue

```diff
@@ -208,8 +215,28 @@ export default {
     this.deselectBlocks()
     this.discoverTallestBlockForDesignerWorkspaceHeight({ aboveTallest: true })
 
+    setTimeout(() => {
```

Comment:
Did you use `setTimeout` to make sure the `document.querySelector()` finds the element ?
If yes, I suggest we use the vue way with `nextTick`
- similar implementation : https://github.com/FLOIP/flow-builder/pull/33/files#diff-43f7c5f1a1252fa694baadbe4e2a0ec2f48c3a9041968f25ed1cfab7b4bdff9aR166
- official doc: https://vuejs.org/v2/api/#Vue-nextTick

Reply:
`setTimeout()` had to be added to wait for the child components to be loaded. If the child components are not loaded then we are trying to fetch an element before it is even created. The `nextTick` did not work in this scenario.